lib/APIPlugin.js

```diff
@@ -201,6 +201,39 @@ class APIPlugin {
 								)
 								.setRange(expr.range)
 						);
+
+					parser.hooks.expression
+						.for("__webpack_module_id__")
```

Comment:
maybe `__webpack_module__.id` this looks more natural + we could want add another property like `__webpack_module__[property]`

Reply:
yep